Can't build when importing fontawesome

My code imports fontawesome with:
```
import "@fortawesome/fontawesome-free/css/all.css";
import "@fortawesome/fontawesome-free/js/all.js";
```
It works well in dev mode with `npm run vite` but fails to build:
```
 sylvain  …  src  main  vue  npm run build

> wingman.www@0.0.0 build /home/sylvain/git/com.enata.wingman.www/src/main/vue
> vite build

vite v0.16.9
⠸ Building for production...
[vite] Build errored out.
[Error: ENOENT: no such file or directory, open '/home/sylvain/git/com.enata.wingman.www/src/main/vue/node_modules/@fortawesome/fontawesome-free/webfonts/fa-brands-400.eot?#iefix'] {
  errno: -2,
  code: 'PLUGIN_ERROR',
  syscall: 'open',
  path: '/home/sylvain/git/com.enata.wingman.www/src/main/vue/node_modules/@fortawesome/fontawesome-free/webfonts/fa-brands-400.eot?#iefix',
  pluginCode: 'ENOENT',
  plugin: 'vite:css',
  hook: 'transform',
  id: '/home/sylvain/git/com.enata.wingman.www/src/main/vue/node_modules/@fortawesome/fontawesome-free/css/all.css',
  watchFiles: [
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/index.html',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/static/favicon.ico',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/main.js',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/App.vue',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/dataLink.js',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/App.vue?vue&type=script&lang.js',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/App.vue?vue&type=template&id=7e76c700',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/App.vue?vue&type=style&index=0&lang.css',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/static/icons/sportMode.svg',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/static/icons/stabilityActive.svg',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/static/icons/stabilityPure.svg',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/static/icons/joystick.svg',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/static/icons/throttle.svg',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/static/logo-white.svg',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/static/icons/oil-level-error.svg',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/node_modules/@fortawesome/fontawesome-free/js/all.js',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/node_modules/@fortawesome/fontawesome-free/css/all.css',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/store/index.js',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/router/index.js',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/node_modules/bootstrap/dist/js/bootstrap.js',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/node_modules/bootstrap/dist/css/bootstrap.min.css',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/node_modules/vue/dist/vue.runtime.esm-bundler.js',
    '\u0000commonjsHelpers.js',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/node_modules/jquery/dist/jquery.js',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/components/ToggleBox.vue',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/components/ToggleBox.vue?vue&type=script&lang.js',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/components/ToggleBox.vue?vue&type=template&id=119f6e85',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/components/ToggleBox.vue?vue&type=style&index=0&lang.css',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/utils/index.js',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/views/Trim.vue',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/views/Dashboard.vue',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/views/Maintenance.vue',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/views/FullDataReporting.vue',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/node_modules/vuex/dist/vuex.esm-bundler.js',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/views/Trim.vue?vue&type=script&lang.js',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/views/Trim.vue?vue&type=template&id=7d53e720',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/views/Trim.vue?vue&type=style&index=0&lang.css',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/views/Dashboard.vue?vue&type=script&lang.js',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/views/Dashboard.vue?vue&type=template&id=3a90207b',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/views/Dashboard.vue?vue&type=style&index=0&lang.css',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/static/icons/engine.svg',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/static/icons/consumption.svg',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/views/Maintenance.vue?vue&type=script&lang.js',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/views/Maintenance.vue?vue&type=template&id=b8f50a52',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/views/FullDataReporting.vue?vue&type=script&lang.js',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/views/FullDataReporting.vue?vue&type=template&id=f1fc5c02&scoped=true',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/views/FullDataReporting.vue?vue&type=style&index=0&id=f1fc5c02&scoped=true&lang.css',
    '\u0000/home/sylvain/git/com.enata.wingman.www/src/main/vue/node_modules/jquery/dist/jquery.js?commonjs-proxy',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/node_modules/vue-router/dist/vue-router.esm-bundler.js',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/components/MomentaryPushButton.vue',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/components/HeadingGauge.vue',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/utils/numpad.css',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/utils/numpad.js',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/components/CircularGauge.vue',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/components/DelayedConfirmation.vue',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/components/SpeedGauge.vue',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/components/PitchGauge.vue',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/components/FuelGauge.vue',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/components/TouchScroller.vue',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/node_modules/popper.js/dist/esm/popper.js',
    '\u0000/home/sylvain/git/com.enata.wingman.www/src/main/vue/node_modules/popper.js/dist/esm/popper.js?commonjs-proxy',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/components/MomentaryPushButton.vue?vue&type=script&lang.js',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/components/MomentaryPushButton.vue?vue&type=template&id=00a3b465',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/components/MomentaryPushButton.vue?vue&type=style&index=0&lang.css',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/components/HeadingGauge.vue?vue&type=script&lang.js',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/components/HeadingGauge.vue?vue&type=template&id=2504278d',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/components/HeadingGauge.vue?vue&type=style&index=0&lang.css',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/components/CircularGauge.vue?vue&type=script&lang.js',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/components/CircularGauge.vue?vue&type=template&id=132e97e2',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/components/CircularGauge.vue?vue&type=style&index=0&lang.css',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/components/PitchGauge.vue?vue&type=script&lang.js',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/components/PitchGauge.vue?vue&type=template&id=73e23f6d&scoped=true',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/components/PitchGauge.vue?vue&type=style&index=0&id=73e23f6d&scoped=true&lang.css',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/components/DelayedConfirmation.vue?vue&type=script&lang.js',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/components/DelayedConfirmation.vue?vue&type=template&id=a04e7538',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/components/DelayedConfirmation.vue?vue&type=style&index=0&lang.css',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/components/SpeedGauge.vue?vue&type=script&lang.js',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/components/SpeedGauge.vue?vue&type=template&id=1e54da3c',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/components/SpeedGauge.vue?vue&type=style&index=0&lang.css',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/components/FuelGauge.vue?vue&type=script&lang.js',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/components/FuelGauge.vue?vue&type=template&id=1d689138',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/components/FuelGauge.vue?vue&type=style&index=0&lang.css',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/components/TouchScroller.vue?vue&type=script&lang.js',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/components/TouchScroller.vue?vue&type=template&id=2df5357c&scoped=true',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/components/TouchScroller.vue?vue&type=style&index=0&id=2df5357c&scoped=true&lang.css',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/node_modules/@vue/runtime-dom/dist/runtime-dom.esm-bundler.js',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/node_modules/suncalc/suncalc.js',
    '/home/sylvain/git/com.enata.wingman.www/src/main/vue/node_modules/d3/index.js'
  ]
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! wingman.www@0.0.0 build: `vite build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the wingman.www@0.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/sylvain/.npm/_logs/2020-05-24T06_59_10_396Z-debug.log
 sylvain  …  src  main  vue  cat /home/sylvain/.npm/_logs/2020-05-24T06_59_10_396Z-debug.log
0 info it worked if it ends with ok
1 verbose cli [
1 verbose cli   '/usr/bin/node',
1 verbose cli   '/home/sylvain/.npm-global/bin/npm',
1 verbose cli   'run',
1 verbose cli   'build'
1 verbose cli ]
2 info using npm@6.14.5
3 info using node@v12.16.3
4 verbose run-script [ 'prebuild', 'build', 'postbuild' ]
5 info lifecycle wingman.www@0.0.0~prebuild: wingman.www@0.0.0
6 info lifecycle wingman.www@0.0.0~build: wingman.www@0.0.0
7 verbose lifecycle wingman.www@0.0.0~build: unsafe-perm in lifecycle true
8 verbose lifecycle wingman.www@0.0.0~build: PATH: /home/sylvain/.npm-global/lib/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:/home/sylvain/git/com.enata.wingman.www/src/main/vue/node_modules/.bin:/home/sylvain/.npm-global/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/home/sylvain/bin:/home/sylvain/bin
9 verbose lifecycle wingman.www@0.0.0~build: CWD: /home/sylvain/git/com.enata.wingman.www/src/main/vue
10 silly lifecycle wingman.www@0.0.0~build: Args: [ '-c', 'vite build' ]
11 silly lifecycle wingman.www@0.0.0~build: Returned: code: 1  signal: null
12 info lifecycle wingman.www@0.0.0~build: Failed to exec build script
13 verbose stack Error: wingman.www@0.0.0 build: `vite build`
13 verbose stack Exit status 1
13 verbose stack     at EventEmitter.<anonymous> (/home/sylvain/.npm-global/lib/node_modules/npm/node_modules/npm-lifecycle/index.js:332:16)
13 verbose stack     at EventEmitter.emit (events.js:310:20)
13 verbose stack     at ChildProcess.<anonymous> (/home/sylvain/.npm-global/lib/node_modules/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)
13 verbose stack     at ChildProcess.emit (events.js:310:20)
13 verbose stack     at maybeClose (internal/child_process.js:1021:16)
13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:286:5)
14 verbose pkgid wingman.www@0.0.0
15 verbose cwd /home/sylvain/git/com.enata.wingman.www/src/main/vue
16 verbose Linux 5.6.13-300.fc32.x86_64
17 verbose argv "/usr/bin/node" "/home/sylvain/.npm-global/bin/npm" "run" "build"
18 verbose node v12.16.3
19 verbose npm  v6.14.5
20 error code ELIFECYCLE
21 error errno 1
22 error wingman.www@0.0.0 build: `vite build`
22 error Exit status 1
23 error Failed at the wingman.www@0.0.0 build script.
23 error This is probably not a problem with npm. There is likely additional logging output above.
24 verbose exit [ 1, true ]
 sylvain  …  src  main  vue  
```

### package.json:
```
{
  "name": "wingman.www",
  "version": "0.0.0",
  "scripts": {
    "dev": "vite",
    "build": "vite build"
  },
  "dependencies": {
    "@fortawesome/fontawesome-free": "^5.13.0",
    "bootstrap": "4.5.0",
    "d3": "^5.16.0",
    "jquery": "^3.5.1",
    "popper.js": "^1.16.1",
    "suncalc": "^1.8.0",
    "vue": "^3.0.0-beta.14",
    "vue-router": "^4.0.0-alpha.12",
    "vuex": "^4.0.0-beta.2"
  },
  "devDependencies": {
    "@vue/compiler-sfc": "^3.0.0-beta.14",
    "vite": "^0.16.9"
  }
}
```

### main.js:
```
import {createApp} from "vue";
import App from "./App.vue";
import store from "./store";
import router from "./router";

import "jquery";
import "bootstrap";
import "bootstrap/dist/css/bootstrap.min.css";
import "@fortawesome/fontawesome-free/css/all.css";
import "@fortawesome/fontawesome-free/js/all.js";

import "./dataLink";

const app = createApp( App );
app.use( store );
app.use( router );
app.mount( '#app' );
```

## System Info
- `vite` version: 0.16.9
- Operating System: Fedora 32
- Node version: 12.6.3

--- a/playground/test-assets/TestAssets.vue
+++ b/playground/test-assets/TestAssets.vue
@@ -44,7 +44,7 @@ export default {
   font-style: italic;
   font-weight: 400;
   font-display: swap;
-  src: url('../fonts/Inter-Italic.woff2') format('woff2'),
+  src: url('../fonts/Inter-Italic.woff2?#iefix') format('woff2'),
     url('/fonts/Inter-Italic.woff') format('woff');
 }
 

--- a/src/node/build/buildPluginAsset.ts
+++ b/src/node/build/buildPluginAsset.ts
@@ -1,7 +1,7 @@
 import path from 'path'
 import fs from 'fs-extra'
 import { Plugin, OutputBundle } from 'rollup'
-import { isStaticAsset } from '../utils'
+import { cleanUrl, isStaticAsset } from '../utils'
 import hash_sum from 'hash-sum'
 import slash from 'slash'
 import mime from 'mime-types'
@@ -24,6 +24,7 @@ export const resolveAsset = async (
   assetsDir: string,
   inlineLimit: number
 ): Promise<AssetCacheEntry> => {
+  id = cleanUrl(id)
   const cached = assetResolveCache.get(id)
   if (cached) {
     return cached
@@ -48,7 +49,7 @@ export const resolveAsset = async (
     const resolvedFileName = `${baseName}.${hash_sum(id)}${ext}`
 
     let url = slash(path.join(publicBase, assetsDir, resolvedFileName))
-    const content = await fs.readFile(id)
+    const content = fs.readFileSync(id)
     if (!id.endsWith(`.svg`) && content.length < Number(inlineLimit)) {
       url = `data:${mime.lookup(id)};base64,${content.toString('base64')}`
     }

--- a/src/node/build/buildPluginAsset.ts
+++ b/src/node/build/buildPluginAsset.ts
@@ -49,7 +49,7 @@ export const resolveAsset = async (
     const resolvedFileName = `${baseName}.${hash_sum(id)}${ext}`
 
     let url = slash(path.join(publicBase, assetsDir, resolvedFileName))
-    const content = fs.readFileSync(id)
+    const content = await fs.readFile(id)
     if (!id.endsWith(`.svg`) && content.length < Number(inlineLimit)) {
       url = `data:${mime.lookup(id)};base64,${content.toString('base64')}`
     }

